src/Serilog.Sinks.Console/ConsoleLoggerConfigurationExtensions.cs

```diff
@@ -52,12 +53,13 @@ public static class ConsoleLoggerConfigurationExtensions
             IFormatProvider formatProvider = null,
             LoggingLevelSwitch levelSwitch = null,
             LogEventLevel? standardErrorFromLevel = null,
-            ConsoleTheme theme = null)
+            ConsoleTheme theme = null, 
+            bool applyThemeOnOutputRedirection = false)
```

Comment:
`applyThemeToRedirectedOutput`?

Reply:
Even better! I'll change the parameter name.